### ISSUE TITLE
Decide code style for VCS and CDMS (at minimum)

### ISSUE THREAD

**Issue by aashish24** [UV-CDAT/uvcdat#1603]
@sankhesh @doutriaux1 @chaosphere2112 @dnadeau4 its really bothering me that we don't have a consistent style. A lack of consistent style leads to bugs and difficult in reading code. We should discuss it post 2.4.

**Comment by doutriaux1** [comment]
@aashish24 I agree, but whatever style we decide in we will have to keep the old names in place as duplicate we can't go ahead and break 25years worth of script.
